base/deprecated.jl

```diff
@@ -1632,6 +1632,12 @@ function SymTridiagonal(dv::AbstractVector{T}, ev::AbstractVector{S}) where {T,S
     SymTridiagonal(convert(Vector{R}, dv), convert(Vector{R}, ev))
 end
 
+# issue #22791
+@deprecate_binding select partialsort
```

Comment:
don't you only need `@deprecate_binding` for types/modules? isn't `@deprecate` good enough for functions?